app/helpers/dashboard/protocols_helper.rb

```diff
@@ -47,7 +47,15 @@ def pis_display(protocol)
     protocol.principal_investigators.map(&:full_name).join ", "
   end
 
-  def archived_button_display(protocol)
-    content_tag( :button, (protocol.archived ? t(:dashboard)[:protocols][:table][:unarchive] : t(:dashboard)[:protocols][:table][:archive])+" #{protocol.type.capitalize}", type: 'button', class: 'protocol-archive-button btn btn-warning btn-sm' )
+  def display_requests_button(protocol, admin_protocols, current_user)
+    if !protocol.sub_service_requests.empty? && (!protocol.project_roles.find_by(identity: current_user).nil? || admin_protocols.try(:include?, protocol.id))
+      content_tag( :button, t(:dashboard)[:protocols][:table][:requests], type: 'button', class: 'requests_display_link btn btn-default btn-sm' )
```

Comment:
This is a little hard to follow

    (!protocol.project_roles.find_by(identity: current_user).nil?